src/app/screens/Home/index.tsx

```diff
@@ -248,7 +248,11 @@ function Home() {
 
                   if (lnData[0].method === "lnurl") {
                     const lnurl = lnData[0].address;
-                    const lnurlDetails = await lnurlLib.getDetails(lnurl); // throws if invalid.
+                    const lnurlDetails = await lnurlLib.getDetails(lnurl);
+                    if ("status" in lnurlDetails && "reason" in lnurlDetails) {
```

Comment:
I would check for `status == error` because what `status` can also be `ok`.
So to me this check does not make sooo much sense. 
and the `reason` property also sounds indirect.

but I don't know why we talk so much about this :)